### ISSUE TITLE
position of the chapter title prompt

### ISSUE THREAD

**Issue by AstudilloMarillo** [mar04/chapters_for_mpv#13]
I'm a bit noob at editing these scripts. I want to change the position of the chapter title prompt from bottom left to top left, since it's kinda hidden behind my playback control buttons. Could you maybe help me out with which lines I'm supposed to change in the chapters.lua file to achieve this?

**Comment by mihawk90** [comment]
As far as I know the script has no control over this.
Make sure you are on the latest version of the script because some things have changed regarding input.

If the input is covered by the player controls it sounds like you are still using a version that's relying on the `mpv-user-input` script, because I had the same issue on it.
Current versions rely on MPV's native input module and from what I can tell it makes sure not to cover any other UI elements:

![Image](https://github.com/user-attachments/assets/554cb688-45a9-4872-bb81-fdc11b1626c4)

**Comment by AstudilloMarillo** [comment]
Thanks for the response, I'll try out the new version of the script.

**Comment by mar04** [comment]
Yes, the script has no control over the prompt. The appearance of the prompt is controlled by mpv's built in console module, its default options are [here](https://github.com/mpv-player/mpv/blob/07571859ac28f4b9c048680a4fb454d146040cb1/player/lua/console.lua#L30), and they can be overwritten by creating a config file `~/.config/mpv/script-opts/console.conf`
e.g.

`font_size=40`
